Framework/Interaction/Tax.php

```diff
@@ -685,6 +691,15 @@ public function getGetTaxRequestForSalesObject($object) {
 
         $customer = $this->getCustomerById($order->getCustomerId());
         $customerUsageType = $customer ? $this->taxClassHelper->getAvataxTaxCodeForCustomer($customer) : null;
+
+        $orderIncrementId = '';
+        try {
+            $order = $this->orderRepository->get($object->getOrderId());
+            $orderIncrementId = $order->getIncrementId();
+        } catch (\Magento\Framework\Exception\NoSuchEntityException $e) {
```

Comment:
Need to make it generalise in a catch block instead of using Megento framework Exception.